### ISSUE TITLE
Nesting Depth does not detect selectors on the same line

### ISSUE THREAD

**Issue by acarnwath** [sasstools/gulp-sass-lint#85]
This throws the warning correctly: 
```
.super {
	.deep {
		.selector {
			.that {
				.just .keeps .going {
					color: red;
				}
			}
		}
	}
}
```

`Nesting depth 4 greater than max of 3`

This does not:
```
.super .deep .selector .that .just .keeps .going {
	color: red;
}
```

I would expect both of them to throw the error, since the output is the same.
